File: cdk/src/main/java/com/myorg/CdkApp.java
package com.myorg;

import software.amazon.awscdk.core.App;

import java.util.Arrays;

public class CdkApp {
    public static void main(final String[] args) {
        App app = new App();

        new CdkStack(app, "CdkStack");

        app.synth();
    }
}

File: cdk/src/main/kotlin/com/myorg/CdkApp.kt
package com.myorg

import software.amazon.awscdk.core.App

object CdkApp {
    @JvmStatic
    fun main(args: Array<String>) {
        val app = App()
        CdkStack(app, "CdkStack")
        app.synth()
    }
}